flowetl/mounts/config/etl/calls/extract.sql

```diff
@@ -1,2 +1,2 @@
-DROP TABLE IF EXISTS {{ extract_table }};
-CREATE TABLE {{ extract_table }} (LIKE events.{{cdr_type.name.lower()}} INCLUDING ALL)
+DROP TABLE IF EXISTS {{ dag_run.conf.extract_table }};
```

Comment:
Out of interest, is the purpose of adding the `dag_run.conf` prefix to make the code more explicit / readable?

Reply:
I think I'd prefer using the [built in macros](https://airflow.apache.org/macros.html) or at a push, user defined macros. Most of the names are (or should be) variations on the type of thing being etl'd combined with the date, no?

Reply:
The `dag_run.conf` prefix was needed to get the vars from the template config - will look into using the macros, the extract and transform tables have an uuid (the same one) appended to them, but this _should_ be possible using `macros.uuid` - there might be some things need adding for the `clean` task to then work correctly.

Reply:
Hmm, I'm in two minds about whether I agree with this or not. Are you suggesting to replace e.g. `{{ extract_table }}` with (some variation of) `etl.x{{ uuid_sans_underscore }}`? I find the former _much_ more meaningful and easy to understand at a glance.

Granted, if you need to debug something you need to look up the definition of `extract_table`, but 99% of the time you don't care about what the extract table is called but you _do_ want to know that you're dealing with the `extract_table` rather than, say, `transform_table`, no? Aren't `{{ extract_table }}` and `{{ transform_table }}` much more readable than `etl.x{{ uuid_sans_underscore }}` and `etl.t{{ uuid_sans_underscore }}`?

Reply:
Finding an alternative to the uuid suffix would also seem fine to me tbh.

Reply:
I see your point there @maxalbert - I was thinking something like `etl.{{ cdr_type }}_{{ ds_nodash }}`, but I'd also be happy with `extract_table` being a custom macro that does that or similar.

I just really dislike stuff being hidden in the `conf` ;)

Reply:
Sure, but no matter how you determine the names of these tables, as a reader of the code (and/or someone who needs to drop in their own ETL snippets), wouldn't you rather know what their _purpose_ is rather than what they're called? I feel it creates less mental overhead because you don't need to worry about coming up with a good name / temporary table location yourself and can just use the one that flowetl has already helpfully provided you with. Or do you feel differently?

Reply:
(Sorry, that was a response to your previous comment - somehow Github doesn't refresh the page while I'm writing a reply. 😂)

And yes, I take your point about stuff being hidden in the conf, although I don't mind too much because at least it makes it obvious where to look if you need to look up the value. (Incidentally, needing to use the conf to look up those values is yet another reason for me to not like Airflow too much. ;))

Reply:
I suggest to leave this out of the current PR and _if_ we want to change something around this address it in a separate PR.